themes/default/content/learn/abstraction-encapsulation/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Abstraction and Encapsulation"
+layout: module
+date: 2021-11-17
+draft: true
+description: |
+    Explore abstraction and encapsulation with Pulumi programs using component
+    resources.
+meta_desc: |
+    Explore abstraction and encapsulation with Pulumi programs using component
+    resources.
+index: 6
+# meta_image: meta.png
+level: intermediate
+youll_learn:
+    - Making reusable abstractions
+    - Encapsulating Pulumi components
+    - Building your own resources
+tags:
+    - learn
+    - components
+providers:
+    - aws
+block_external_search_index: true
+---
+
+## Time
+
+How long this tutorial will take depends on your internet connection, reading
```

Comment:
Sorry, old holdover from another company's markdown policies; it was easier for editing at the time and we didn't have it in a style guide here (though I see that's been added since this PR went in).

Reply:
Not applying this change; this is standard language.